composer.json

```
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.1 || ^8.0",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2 || ^0.7",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
```

I'd do 
```suggestion
        "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2 || ^0.7.1",
```

to ensure at least the patch fix.